openassessment/xblock/staff_info_mixin.py

```diff
@@ -256,6 +276,15 @@ def get_student_info_path_and_context(self, student_id):
             peer_assessments = peer_api.get_assessments(submission_uuid)
             submitted_assessments = peer_api.get_submitted_assessments(submission_uuid, scored_only=False)
 
+            # Get the data we need for instructor override of the student's score
+            rubric_dict = create_rubric_dict(self.prompt, self.rubric_criteria_with_labels)
+            scores = peer_api.get_data_for_override_score(
+                submission_uuid,
+                student_item,
+                rubric_dict,
+            )
+            problem_closed, _, _, _ = self.is_closed(step='peer-assessment', course_staff=False)
```

Comment:
To avoid shadowing `_`/`gettext`, let's use the pylint `dummy` convention here:
```python
problem_closed, dummy0, dummy1, dummy2 = self.is_closed(step='peer-assessment', course_staff=False)